package module10.TestExceptionsAndErrors;

/**
 * НЕ ЗАПУСКАТЬ!!!
 * БУДЕТ БЕСКОНЕЧНО ЛОВИТЬ StackOverflowError. РАБОТАЕТ ПО ТИПУ РЕКУРСИИ
 */
public class ExceptionInfinity {
    public static void main ( String[] args ) {
//        try{
//            main ( args );
//        } catch (StackOverflowError e) {
//            main ( args );
//        }


//
//        try{
//            main ( args );
//        } catch (StackOverflowError e) {
//            main ( args );
//        } finally {
//            main ( args );
//        }



//        try{
//            main ( args );
//        } finally {
//            main ( args );
//        }



    }
}
